ethereum/forge-test/query/QueryTest.sol

```diff
@@ -0,0 +1,256 @@
+// SPDX-License-Identifier: Apache 2
+
+// forge test --match-contract QueryTest
+
+pragma solidity ^0.8.4;
+
+// @dev QueryTest is a library to build Cross Chain Query (CCQ) responses for testing purposes.
+abstract contract QueryTest {
```

Comment:
This might be more useful as a library rather than an abstract contract (since all functions are pure). That way you could use it within deployed contracts as well if needed.

Reply:
Converted it to a library.

Reply:
Converted it to a library.